src/sentry/static/sentry/app/views/dashboards/utils/isTimeSeries.tsx

```diff
@@ -0,0 +1,8 @@
+type Query = {
+  groupby: string[];
+};
+
+// Consider a query a time series if
```

Comment:
these can be converted to `/* */` comments so jsdoc picks them up as descriptions, but here its not that useful anyway

Reply:
Good to know 👍